apps/app/src/services/renderer/remark-plugins/plantuml.ts

```diff
@@ -1,15 +1,42 @@
 import plantuml from '@akebifiky/remark-simple-plantuml';
 import { Plugin } from 'unified';
+import { Parent, Node } from 'unist';
 import urljoin from 'url-join';
 
 type PlantUMLPluginParams = {
   plantumlUri: string,
-}
+};
 
 export const remarkPlugin: Plugin<[PlantUMLPluginParams]> = (options) => {
   const plantumlUri = options.plantumlUri;
+  const baseUri = urljoin(plantumlUri);
```

Comment:
Do not need `/svg` join?
`const baseUrl = urljoin(plantumlUri, '/svg');`

Reply:
Sorry for my lack of consideration. 
I think it is necessary to specify the type of the resulting image with plantuml instead of using the default (png) type.
So I modified the code and specified the type of the generated image.
I also changed `baseUri `-> `baseUrl` because that is also necessary